src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -77,11 +85,21 @@ public class SnowflakeSinkTask extends SinkTask {
   // check connect-distributed.properties file used to start kafka connect
   private final int rebalancingSleepTime = 370000;
 
-  private static final LoggerHandler LOGGER = new LoggerHandler(SnowflakeSinkTask.class.getName());
+  // the dynamic logger is intended to be attached per task instance. the instance id will be set
+  // during task start,
+  // however if it is not set, it falls back to the static logger
+  private static final LoggerHandler STATIC_LOGGER =
+      new LoggerHandler(SnowflakeSinkTask.class.getName() + "_STATIC");
+  private LoggerHandler DYNAMIC_LOGGER;
+
+  private long taskStartTime;
 
   /** default constructor, invoked by kafka connect framework */
   public SnowflakeSinkTask() {
     // nothing
```

Comment:
nit: remove nothing

Reply:
I'm confused, remove... the blank line? Not sure what 'remove nothing' means

Reply:
never mind i see the nothing comment lol